geopandas/tests/util.py

```diff
@@ -13,6 +13,15 @@
 HERE = os.path.abspath(os.path.dirname(__file__))
 PACKAGE_DIR = os.path.dirname(os.path.dirname(HERE))
 
+_TEST_DATA_DIR = os.path.join(PACKAGE_DIR, "geopandas", "tests", "data")
+_NYBB = "zip://" + os.path.join(_TEST_DATA_DIR, "nybb_16a.zip")
+_NATURALEARTH_CITIES = os.path.join(
+    _TEST_DATA_DIR, "naturalearth_cities", "naturalearth_cities.shp"
+)
+_NATURALEARTH_LOWRES = os.path.join(
+    _TEST_DATA_DIR, "naturalearth_lowres", "naturalearth_lowres.shp"
+)
```

Comment:
I suppose we can use geodatasets in benchmarks to avoid this.

Reply:
As long as the caching in geodatasets won't mess with the benchmarks? I imagine it would be fine once each dataset exists in the local cache, just that there might be a small slowdown in the benchmarks after this PR?